### ISSUE TITLE
Release Binaries

### ISSUE THREAD

**Issue by flywire** [coherentgraphics/cpdf-binaries#87]
If https://github.com/coherentgraphics/cpdf-binaries/releases contained the binaries it would save users from downloading all and extracting the required binary.

**Comment by flywire** [comment]
This might be useful: https://github.com/skx/github-action-publish-binaries

**Comment by GitHubRulesOK** [comment]
@johnwhitington @flywire It is possible to download the files from the online folder,
Personally can only use thus need to download https://github.com/coherentgraphics/cpdf-binaries/blob/master/Windows32bit/cpdf.exe

HOWEVER that throws my system into disarray Black screen of Half Death and then luckily (unlike Firefox needing a reboot !) Defender says 
>Windows protected your PC
Microsoft Defender SmartScreen prevented an unrecognised app from starting. Running this app might put your PC at risk.
[More info](about:blank#)

It would be the same as pull from zip without saying the zip is to be unblocked.

Bottom line after download either way I need to be aware to unblock the download before it runs into Defender Mode.
